### PR TITLE
New shop

### PR DESCRIPTION
**Pull request by MrTanoshii**
DO NOT MERGE UNTIL REVIEW

### CODE REVIEW

**Comment by MrTanoshii** [comment]
#61

File: gameview.py [hunk 1]
```diff
@@ -320,6 +323,7 @@ def check_collisions(self):
 
                     # Play enemy death sfx
                     Audio.play_rand_sound(enemy.sfx_death_list)
+                    self.enemy_killed += 1  # Increment Enemy killed
```
Comment (Jeb4dev) [inline gameview.py:326]: I don't think we want the shop to open automatically. I would remove this and just use the one inside map menu.

Comment (Jeb4dev) [reply gameview.py:326]: also lines 66 - 68

File: shopview.py [hunk 2]
```diff
@@ -1,34 +1,96 @@
...
+
+        # Clear the screen to the background color
+        self.clear()
+
+        # arcade.draw_lrwh_rectangle_textured(0, 0,
+        #                                     C.SCREEN_WIDTH, C.SCREEN_HEIGHT,
+        #                                     self.background)
+
+        self.move_spritel.draw(pixelated=True)
+        self.move_spriter.draw(pixelated=True)
```
Comment (Jeb4dev) [inline shopview.py]: have these assets already pixelated, also i am not sure if i like the white bg

File: shopview.py [hunk 3]
```diff
@@ -1,34 +1,96 @@
...
-    def on_draw(self):
-        """Draw the menu"""
-        self.clear()
+    def on_mouse_press(self, x, y, button, modifiers):
+
+        if self.move_spritel.collides_with_sprite(self.cursor_sprite):
+            self.c_p = (self.c_p + 1)
+            if self.c_p > len(self.list1):
+                self.c_p = 0
+            self.Player1 = PreviewSprite(self.list1[self.c_p])
```
Comment (Jeb4dev) [inline shopview.py]: this line causes an error

in certain index / image

File: shopview.py [hunk 4]
```diff
@@ -1,34 +1,96 @@
...
+
+        self.center_x = 650
+        self.center_y = 400
+        self.scale = 1
+        self.cur_texture = 0
+
+        base_path = f"assets/{dir}/"
+
+        self.texture_list = []
+        for filename in os.listdir(f"{base_path}"):
```
Comment (Jeb4dev) [inline shopview.py:101]: maybe better way to do this, in certain order. also how u define prices etc.
maybe write filepaths in configs
also scales differ, so in config write scale for everything

**Review by Jeb4dev — CHANGES_REQUESTED** [review]
check my comments in files / lines
also where are shop_frames used, why 4 image that look identical

also needs to add:
- prices
- buy button
- names
- consistant scaling
- maybe write how buing thing makes it better

anyways I like how the animation in in the middle animating

File: shopview.py (hunk 3 above)
Comment (MrTanoshii) [reply shopview.py]: Fixed in 6e4f722eae0c53c23b8a5e76561c856bcc557b93

**Comment by MrTanoshii** [comment]
#111

**Pull request closed without merging**